python/tests/hypothesis/arcticdb/test_sort_merge.py

```diff
@@ -1,71 +1,229 @@
-from re import S
 import pandas as pd
 import pytest
-from hypothesis import assume
+from hypothesis import assume, given, settings
 import hypothesis.extra.pandas as hs_pd
 import hypothesis.strategies as st
 from hypothesis.stateful import RuleBasedStateMachine, rule, initialize, run_state_machine_as_test, precondition
 from collections import namedtuple
 from pandas.testing import assert_frame_equal
 from arcticdb.version_store.library import StagedDataFinalizeMethod
+from arcticdb.exceptions import UserInputException, StreamDescriptorMismatch, UnsortedDataException, NoSuchVersionException, SchemaException
+import numpy as np
+import string
+from arcticdb.util._versions import IS_PANDAS_TWO
+from pandas.api.types import is_numeric_dtype, is_integer_dtype, is_float_dtype
+from arcticdb.util.hypothesis import use_of_function_scoped_fixtures_in_hypothesis_checked
+from arcticdb.toolbox.library_tool import KeyType
 
 ColumnInfo = namedtuple('ColumnInfo', ['name', 'dtype'])
 
-COLUMN_DESCRIPTIONS = [ColumnInfo("a", "float"), ColumnInfo("b", "int64"), ColumnInfo("c", "str"), ColumnInfo("d", "datetime64[ns]")]
+COLUMNS = [f"col_{i}" for i in range(0, 5)]
```

Comment:
is it possible to have an unsigned type and a few more columns? Five is very narrow.

Reply:
Do you have any number on your mind? The only concern is that generating too many columns might slow the tests but we can play with it until we're happy.